aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/AWSS3StoragePlugin.java

```diff
@@ -309,6 +309,7 @@ public StorageGetUrlOperation<?> getUrl(
         return getUrl(path, StorageGetUrlOptions.defaultInstance(), onSuccess, onError);
     }
 
+    @OptIn(markerClass = InternalAmplifyApi.class)
```

Comment:
This opt in can be removed once removing annotation from the Request class as mentioned.

Reply:
annotation removed